timetracker/entries/views.py

```diff
@@ -1,30 +1,25 @@
 from django.shortcuts import render, get_object_or_404
+from django.core.urlresolvers import reverse
 
 from .forms import EntryForm, ProjectForm, ClientForm
 from .models import Client, Entry, Project
 
+from django.views.generic import CreateView, ListView, UpdateView, RedirectView
 
-def clients(request):
-    if request.method == 'POST':
-        # Create our form object with our POST data
-        form = ClientForm(request.POST)
-        if form.is_valid():
-            # If the form is valid, create a client with submitted data
-            # Below is the shortcut equivalent of:
-            # client = Client()
-            # client.name = form.cleaned_data['name']
-            # client.save()
-            # Sometimes you don't want to save the object until the end, 
-            # sometimes you don't care!
-            client = Client.objects.create(name=form.cleaned_data['name'])
-    else:
-        form = ClientForm()
+class ClientCreateView(CreateView):
+    """
+    View to create a Client instance every
+    time the form is submitted
+    """
+    model = Client
+    template_name = 'clients.html'
+    form_class = ClientForm
+    success_url = '/clients/'
```

Comment:
You can use `django.core.urlresolvers.reverse_lazy()` to not write hard-coded paths